File: hw/Rodriguez/u2/hw18/Zoo/ec_edu_espe_zoo_model/shark.py
from .fish import Fish
from datetime import date
from .cage import Cage

class Shark(Fish):
    def __init__(self, id, description, date_onborn, gender, number_of_bones, cage):
        super().__init__(id, description, date_onborn, gender, number_of_bones, cage)

    def feed(self, food):
        print("Feeding to Shark")

File: ws/Vaca/u2/WS24Zoo/Zoo/Zoo/src/main/java/ec/edu/espe/zoo/model/Shark.java
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author Luis Vaca, DCCO-ESPE
 */
public class Shark extends Fish{

    public Shark(int id, String description, Date dateOnborn, char gender, int numberOfBones, Cage cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

    @Override
    public void Feed(int food) {
        System.out.println("Feeding to Shark ");
    }
    
}
